### ISSUE TITLE
Duda sobre el formato de entrega del control 1

### ISSUE THREAD

**Issue by gabo1243** [IIC2413/Syllabus-2023-1#4]
Hola, sobre el control: ¿debe hacerse en LateX o algo asi? ¿O puedo hacerlo en una hoja escrita y luego tonarle una foto para pasarlo a pdf?

Gracias de antemano.

**Comment by VicenteVicente** [comment]
Hola Gabriel!!

La entrega es en PDF. En caso de que decidan hacerlo en una hoja escrita, asegurense de que su respuesta quede **clara y legible** (esto último queda bajo su responsabilidad).

Éxito!

**Comment by VicenteVicente** [comment]
@czamoraz claro, puedes responder las preguntas inmediatamente debajo de ellas. Para esta instancia no entregaremos el archivo `.tex`. Sin embargo, te dejo este link con una plantilla con el formato UC https://github.com/open-source-uc/latex-template

**Comment by czamoraz** [comment]
Gracias a ambos!
